addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -114,10 +116,12 @@ var KanbanView = BasicView.extend({
      */
     _updateMVCParams: function () {
         this._super.apply(this, arguments);
-        var defaultGroupBy = this.arch.attrs.default_group_by;
-        this.loadParams.groupBy = defaultGroupBy ?
-                                    [defaultGroupBy] :
-                                    (this.loadParams.groupedBy || []);
+        if (!this.noDefaultGroupby) {
```

Comment:
Do you remember for which kanban view you did this?

Reply:
@adr-odoo We did it for reason, we do not want to open Grouped by kanban in selection mode, consider the case where project task many2one is used, note that project task's many2one has default grouped view, when kanban is opened it should always show normal kanban view, not grouped one, so ```noDefaultGroupby``` is passed ```true``` from view_dailogs.js